.changeset/fast-fireants-fix.md

```diff
@@ -0,0 +1,6 @@
+---
+'@finos/legend-studio': patch
+'@finos/legend-studio-plugin-query-builder': patch
+---
+
+Add V1_getExtraMappingExecutionClasses to send extra plugin elements as part of query execution call.
```

Comment:
`V1_getExtraMappingExecutionClasses`

Reply:
Yeah, let's give a bit of background here on why we need this.